dependencies.gradle

```diff
@@ -46,6 +47,7 @@ ext {
     navigationVersionName = "1.0.0-alpha3"
     testingVersionName = "1.0.0-alpha4"
     snackbarVersionName = "1.0.0-alpha2"
+    navigationRailVersionName = '1.0.0-beta1'
```

Comment:
Since it's the first release of the library, it could make sense to have it as `alpha1` so if we need to make changes on the API, that's fine.

Reply:
Done